src/Pipfile.lock

```diff
@@ -1371,78 +1373,78 @@
                 "sha256:1421ebfc7648a39a5c58c601b154165d05cf47a3cd0ccb70857cbdacf6c8f2b8",
                 "sha256:b863f8ff057c522164b6067c9e28b041161b4be5ba4d0daceeaa50a163822d3c"
             ],
-            "markers": "python_version >= '3.6'",
+            "index": "pypi",
             "version": "==2.0.3"
```

Comment:
weird, werkzeug was already pinned here. I think we just needed to re-lock this without adding this to the Pipfile?

Reply:
I believe the reason why this issue hasn't appeared for you yet is because the last time this lockfile was generated was before a > 2.0.3 version of werkzeug was published (see https://pypi.org/project/Werkzeug/#history)